src/SFML/Graphics/Image.cpp

```diff
@@ -154,7 +155,7 @@ void Image::resize(Vector2u size, Color color)
         }
 
         // Commit the new pixel buffer
-        m_pixels.swap(newPixels);
```

Comment:
> // Create a new pixel buffer first for exception safety's sake

This comment is somewhat tangential to the goal of your PR, but I read this comment earlier this year when working on something else and couldn't make sense of it. What about making a new buffer first has any affect on exception safety?

Reply:
Uhhh... I guess the idea was that if the memory allocation of `newPixels` ends up throwing an exception, the state of `Image::m_pixels` would be unchanged. 

Probably the initial implementation was something like `m_pixels.resiez(static_cast<std::size_t>(size.x) * static_cast<std::size_t>(size.y) * 4)`, but that should have the strong exception guarantee anyway, so I guess we could just do that?

Reply:
I would prefer a simpler implementation using `.resize` instead of doing a dance with extra intermediate vectors.

Reply:
Unless we are missing something else, I agree. I'd be happy to review a PR for that.